helm/resource_release.go

```diff
@@ -537,7 +543,18 @@ func resourceReleaseCreate(ctx context.Context, d *schema.ResourceData, meta int
 	client.CreateNamespace = d.Get("create_namespace").(bool)
 
 	if cmd := d.Get("postrender.0.binary_path").(string); cmd != "" {
-		pr, err := postrender.NewExec(cmd)
+		av := d.Get("postrender.0.args")
+
```

Comment:
We can remove the spaces above and below here so the formatting is similar to the other places do this